theano/gpuarray/basic_ops.py

```diff
@@ -293,6 +293,37 @@ class GpuKernelBase(object):
     """
     params_type = gpu_context_type
 
+    def get_params(self, node):
+        # Default implementation, suitable for most sub-classes.
+        # To be necessarly overridden if a subclass uses a ParamsType
+        # (see GpuCumOp, GpuAdvancedIncSubtensor1_dev20 for examples).
```

Comment:
Just to make sure, I suppose this is going to be added in another split of the original PR?
For now, this one removes the definition of `get_params` in `GpuCumOp`.